server/routes/animeListItem.js

```diff
@@ -69,4 +69,20 @@ const animeItem = req.params.id;
   }
 })
 
+//concept: remove row in user_anime_list based on id
+//user to remove item from their list
+router.put('/:id', async (req, res) => {
+  //delete anime from user_anime_list based on id
+const animeItem = req.params.id;
+const content = req.body.complete;
+
+  try {
+    await db.none(`UPDATE user_anime_list SET complete = true WHERE id=$1`, [animeItem]);
+    res.send({status:'success'});
+  } catch (e) {
+    console.log(e);
+    return res.status(400).json(e);
+  }
+})
+
 export default router;
```

Comment:
Don't forget your new line at the end of files!

Reply:
This is a comment eslint style rule: https://airbnb.io/javascript/#whitespace--newline-at-end. Google around to see why this is a recommendation.

Probably not the most important thing right now unless you are going to set up some automated linting, but it is a good thing to be aware of.